src/routes/api/profile/experience/academic-experience/delete/+server.ts

```diff
@@ -0,0 +1,20 @@
+import { BACKEND_BASE_URL } from '$env/static/private'
+import { json } from '@sveltejs/kit'
+
+export interface AcademicExperienceDeletePayload {
+  id: number
+}
+
+export async function POST({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```

Comment:
```suggestion
export async function DELETE({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```
¿Por qué esto es un POST?